circle.yml

```diff
@@ -37,6 +37,44 @@ executors:
       PLATFORM: mac
 
 commands:
+  run-e2e-test:
+    parameters:
+      browser:
+        description: browser shortname to target
+        type: string
+      chunk:
+        description: e2e test chunk number
+        type: integer
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          command: npm run test-e2e -- --chunk << parameters.chunk >> --browser << parameters.browser >>
+          working_directory: packages/server
+      - store_test_results:
+          path: /tmp/cypress
+      - store-npm-logs
+
+  run-e2e-tests-chrome:
+    parameters:
+      chunk:
+        description: e2e test chunk number
+        type: integer
+    steps:
+      - run-e2e-test:
+          browser: chrome
+          chunk: << parameters.chunk >>
+
+  run-e2e-tests-electron:
+    parameters:
+      chunk:
+        description: e2e test chunk number
+        type: integer
+    steps:
+      - run-e2e-test:
+          browser: electron
+          chunk: << parameters.chunk >>
+
```

Comment:
I think this is too much abstraction and would just prefer to use the base `run-e2e-test` command and pass in `browser` as an option.